content/docs/start/data-management/data-versioning.md

```diff
@@ -35,42 +37,41 @@ $ dvc get https://github.com/iterative/dataset-registry \
 
 <admon type="info">
 
-We use the fancy `dvc get` command to jump ahead a bit and show how a Git repo
-becomes a source for datasets or models — what we call a [data registry].
-`dvc get` can download any file or directory tracked in a <abbr>DVC
+We used `dvc get` above to show how DVC can turn any Git repo into a "[data
+registry]". `dvc get` can download any file or directory tracked in a <abbr>DVC
 repository</abbr>.
 
 [data registry]: /doc/use-cases/data-registry
 
 </admon>
 
-</details>
-
-To start tracking a file or directory, use `dvc add`:
+Use `dvc add` to start tracking the dataset file:
 
 ```cli
 $ dvc add data/data.xml
 ```
 
 DVC stores information about the added file in a special `.dvc` file named
-`data/data.xml.dvc` -- a small text file with a human-readable [format]. This
-metadata file is a placeholder for the original data, and can be easily
-versioned like source code with Git:
+`data/data.xml.dvc`. This small, human-readable metadata file acts as a
+placeholder for the original data for the purpose of Git tracking.
+
+Next, run the following commands to track changes in Git:
 
 ```cli
 $ git add data/data.xml.dvc data/.gitignore
 $ git commit -m "Add raw data"
 ```
 
-The data, meanwhile, is listed in `.gitignore`.
+Now the _metadata about your data_ is versioned alongside your source code,
+while the original data file was added to `.gitignore`.
 
-<details id="add-click-to-see-what-happens-under-the-hood">
+<details id="add-click-to-get-a-peek-under-the-hood">
 
-### 💡 Click to see what happens under the hood.
+### 💡 Expand to get a peek under the hood
```

Comment:
It's technically correct (example, no. 4 [here](https://englishpedia.net/inasentence/peek-in-a-sentence/))
But in terms of what would fit better here: "get a peek" or "take a peek" - not sure. I'm partial to either one. I think "see what happens under the hood" is not a 100% fit because it's losing the "fleeting nature" of a "peek" 😄

Reply:
I see. 
I guess I was not familiar with the variation.

WDYT `take a quick look`?

Again, no strong opinion might be just me not being well versed in the jargon

Reply:
Also no strong opinion, just felt that this would "lighten the mood a bit".
@dberenbaum  - any preference? what would sound more natural?

Reply:
(if this is merged soon I'll followup on that in case we want to change, no objections 😄 )